lib/time.js

```diff
@@ -152,17 +152,27 @@ function CronTime(luxon) {
 			}
 
 			if (typeof this.utcOffset !== 'undefined') {
-				let offset =
-					this.utcOffset >= 60 || this.utcOffset <= -60
-						? this.utcOffset / 60
-						: this.utcOffset;
+				if (typeof this.utcOffset === 'string') {
+					const parts = this.utcOffset.split(':');
+					const hours = parseInt(parts[0]);
+					const minutes = parseInt(parts[1]);
```

Comment:
maybe use `parseInt(parts[1] ?? '0');` in case the colon and minutes are omitted in the string?